ops/model.py

```diff
@@ -281,6 +281,20 @@ def get_secret(self, *, id: Optional[str] = None, label: Optional[str] = None) -
         content = self._backend.secret_get(id=id, label=label)
         return Secret(self._backend, id=id, label=label, content=content)
 
+    def get_cloud_spec(self) -> 'CloudSpec':
+        """Get cloud credential information.
+
+        Access the cloud credential information and return the cloud specification
+        used by the model.
```

Comment:
Maybe refocus this on the cloud specification rather than credentials? Something like:

```suggestion
        """Get details of the cloud in which the model is deployed.

        Returns a specification for the cloud in which the model is deployed, including access credential information.
```